### PR TITLE
Add module to generate download command of current data

### PR DESCRIPTION
**Pull request by sebbousquet**

### CODE REVIEW

**Comment by sebbousquet** [comment]
Fix #143

**Comment by MohamedHamouGisaia** [comment]
Notice of what we discussed : 
- Adding a Copy button
- if Json is chosen, there is a `\` and the end of the command
- the dropdown boxes for sorting are not reset if we check/uncheck fields
- Maybe we can add `&size=9999` to the url to do a pagination of 9999 instead of the default value 10

File: src/app/components/download/download.component.ts [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+   * @param event The step index
+   */
+  public changeStep(event) {
+    if (event.selectedIndex === 2) {
+      this.isCopied = false;
+      this.exportedTypeCommand = this.exportTypeGroup.get('exportType').value;
+      this.orderCommand = '';
+      const filters = new Array<Filter>();
+      this.collaborativeService.collaborations.forEach(element =>
+        filters.push(element.filter)
```
Comment (mbarbet) [inline src/app/components/download/download.component.ts]: Maybe we can use .map function to create filters array ?
const filters = this. collaborativeService.collaborations.map(element=>element.filter)

Comment (sebbousquet) [reply src/app/components/download/download.component.ts]: `this. collaborativeService.collaborations` is a Map, map() operator is not applicable here

**Comment by MohamedHamouGisaia** [comment]
LGTM

**Pull request merged**